packages/javascript-sdk/src/http-client/index.ts

```diff
@@ -167,20 +172,15 @@ abstract class HttpClient extends Dispatcher {
     return headers;
   }
 
-  private static async stepIterator(
-    res: Response,
-    handleStep: HandleStep,
-    type: string,
-    tree: string,
-  ): Promise<void> {
+  private static async stepIterator(res: Response, handleStep: HandleStep): Promise<void> {
     const jsonRes = await res.json();
     const initialStep = new FRStep(jsonRes);
 
     // eslint-disable-next-line no-async-promise-executor
     return new Promise(async (resolve, reject) => {
       async function handleNext(step: FRStep): Promise<void> {
         const input = await handleStep(step);
-        const output = await FRAuth.next(input, { type, tree });
+        const output = await FRAuth.next(input, { tree: '', type: '' });
```

Comment:
I eventually figured out that in order for the transaction approval tree-outcome to take effect, we could only pass the `authIndexType=composite_advice` on the initial authentication request. Having any query parameters for `authIndexType` or `authIndexValue` after the first one, caused a 401 described as "session expired". It was weird but I found a conversation @cerebrl had with others in the team, where this was discovered. I'm fine having this in here, but we may want to discuss why it is necessary for this flow now when previously that may not have been the case - @cerebrl.